static/elements/chromedash-myfeatures-page.js

```diff
@@ -1,36 +1,53 @@
 import {LitElement, css, html, nothing} from 'lit';
-import './chromedash-approvals-dialog';
-import './chromedash-feature-table';
+
 import {SHARED_STYLES} from '../sass/shared-css.js';
 
 
-class ChromedashMyFeatures extends LitElement {
+export class ChromedashMyFeaturesPage extends LitElement {
+  static get styles() {
+    return [
+      ...SHARED_STYLES,
+      css`
+        sl-details {
+          padding: 0 var(--content-padding);
+        }
+      `];
+  }
+
   static get properties() {
     return {
-      signedInUser: {type: String},
-      canEdit: {type: Boolean},
-      canApprove: {type: Boolean},
-      loginUrl: {type: String},
-      starredFeatures: {type: Object}, // will contain a set of starred features
+      user: {type: Object},
+      starredFeatures: {type: Set}, // will contain a set of starred features
+      toastEl: {type: Element},
```

Comment:
Does this need to be a property, or even an instance variable?  The element could be looked up whenever we need to display a toast message.   I think the main idea of a property is that it could change value and trigger the element to re-render, which does not seem to apply here.

Reply:
What you says makes sense, and I removed the toastEl property.

But I'm trying to reduce the amount of `document.querySelector` (especially the toast is used quite often). 

In the new commit, I made a util method `showToastMessage` and import it to the elements. This way we only query once for the toast and at the same time avoid code repetition. Let me know what you think!

Reply:
OK that sounds good.

That reminds me, we should consider replacing chromedash-toast with https://shoelace.style/components/alert?id=toast-notifications.   That might allow us to eliminate one source file.  I am not sure if their component is really better than ours in a meaningful way.